src/pybind/mgr/volumes/fs/operations/template.py

```diff
@@ -35,42 +36,44 @@ def list_snapshots(self):
         """
         raise VolumeException(-errno.ENOTSUP, "operation not supported.")
 
+
 @unique
 class SubvolumeOpType(Enum):
-    CREATE                = 'create'
-    REMOVE                = 'rm'
-    REMOVE_FORCE          = 'rm-force'
-    PIN                   = 'pin'
-    LIST                  = 'ls'
-    GETPATH               = 'getpath'
-    INFO                  = 'info'
-    RESIZE                = 'resize'
-    SNAP_CREATE           = 'snap-create'
-    SNAP_REMOVE           = 'snap-rm'
-    SNAP_LIST             = 'snap-ls'
-    SNAP_INFO             = 'snap-info'
-    SNAP_PROTECT          = 'snap-protect'
-    SNAP_UNPROTECT        = 'snap-unprotect'
-    CLONE_SOURCE          = 'clone-source'
-    CLONE_CREATE          = 'clone-create'
-    CLONE_STATUS          = 'clone-status'
-    CLONE_CANCEL          = 'clone-cancel'
-    CLONE_INTERNAL        = 'clone_internal'
-    ALLOW_ACCESS          = 'allow-access'
-    DENY_ACCESS           = 'deny-access'
-    AUTH_LIST             = 'auth-list'
-    EVICT                 = 'evict'
-    USER_METADATA_SET     = 'user-metadata-set'
-    USER_METADATA_GET     = 'user-metadata-get'
-    USER_METADATA_LIST    = 'user-metadata-ls'
-    USER_METADATA_REMOVE  = 'user-metadata-rm'
-    SNAP_METADATA_SET     = 'snap-metadata-set'
-    SNAP_METADATA_GET     = 'snap-metadata-get'
-    SNAP_METADATA_LIST    = 'snap-metadata-ls'
-    SNAP_METADATA_REMOVE  = 'snap-metadata-rm'
+    CREATE = 'create'
+    REMOVE = 'rm'
+    REMOVE_FORCE = 'rm-force'
```

Comment:
This formatting was deliberately this way for sake of readability.

Reply:
The reason pep8 style guide does not recommend aligning = signs is that sooner or later, you will get a longer variable, which causes you to re-align all those = symbols. It is subjective if aligned = is more readable or not. Given that we have a tracker issue and we want it to be fixed, which means we want to adhere to pep8, I have fixed it. If not we can close the PR.